src/Compilers/CSharp/Portable/Syntax/YieldStatementSyntax.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.CodeAnalysis.CSharp.Syntax
     public partial class YieldStatementSyntax
     {
         public YieldStatementSyntax Update(SyntaxToken yieldKeyword, SyntaxToken returnOrBreakKeyword, ExpressionSyntax expression, SyntaxToken semicolonToken)
-            => Update(attributeLists: default, yieldKeyword, returnOrBreakKeyword, expression, semicolonToken);
+            => Update(attributeLists: AttributeLists, yieldKeyword, returnOrBreakKeyword, expression, semicolonToken);
```

Comment:
nit: it feels like argument labels are no longer necessary for all the arguments changed by this PR, since the argument expression is more descriptive now.

Reply:
I fixed it in the web UI - is there a way to do multiple file changes at once / with a single commit?

Reply:
We'll squash this PR when we merge it, so it'll only be one commit in the history.
I don't know a way of doing multiple changes in a single commit via the web UI.